src/lib/utils/thorchain/memo/addFinalAssetLimitToMemo.ts

```diff
@@ -0,0 +1,70 @@
+import { assertAndProcessMemo } from './assertAndProcessMemo'
+import { MEMO_PART_DELIMITER } from './constants'
+
+type AddLimitToMemoArgs = {
```

Comment:
```suggestion
type AddFinalAssetLimitToMemoArgs = {
```

Reply:
Fixed in [2a6fc6a](https://github.com/shapeshift/web/pull/7146/commits/2a6fc6a57521f17fb2fa9ee67d3d46f222c6e53c)